src/main/java/com/epam/lab/paymentsystem/dao/ConnectionPool.java

```diff
@@ -0,0 +1,68 @@
+package com.epam.lab.paymentsystem.dao;
+
+import javax.naming.NamingException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.LinkedList;
+import java.util.List;
+
+public class ConnectionPool {
+
+    static List<Connection> connectionPool = new LinkedList<Connection>();
+    Connection connect = null;
+
+
+    public static Connection getConnection() throws SQLException, NamingException {
+        Connection connectReturn = null;
+        if (connectionPool.size() < 1) {
+            addConnect();
+            connectReturn = connectionPool.remove(0);
+        } else {
+            connectReturn = connectionPool.remove(0);
+        }
+        return connectReturn;
+    }
+
+    public static void close(Connection connection) {
+        if(connection != null) {
+            connectionPool.add(connection);
+        }
+        else connectionPool.add(new ConnectionPool().connect);
```

Comment:
It looks strange and unclear. I would recommend to use another approach:
```
ConnectionPool pool = new ConnectionPool();
Connection con = pool.getConnection();
connectionPool.add(con);
```
Also please do not forget about `{}` brackets in `if-else` statements